firewall/rule_enforcer.go

```diff
@@ -5,10 +5,10 @@ import (
 	"fmt"
 
 	"github.com/lightninglabs/lightning-terminal/firewalldb"
-	"github.com/lightninglabs/lightning-terminal/perms"
 	mid "github.com/lightninglabs/lightning-terminal/rpcmiddleware"
 	"github.com/lightninglabs/lightning-terminal/rules"
 	"github.com/lightninglabs/lightning-terminal/session"
+	"github.com/lightninglabs/lightning-terminal/subservers"
```

Comment:
Good point, the main idea was for the permission manager to be able to get and returns `SubServerName(s)` typed strings. But it may be worth keeping the permissions where they were + use simple strings

Reply:
ah ok I see - so moving things to avoid the circular dependency. But yeah, perhaps in this case a little bit of copying is better than the large code shift. 

interested to hear what @guggero 's opinion is on this 🙏

Reply:
Yeah, I'd probably also go with having a separate list of the subserver names and keeping the permissions where they are.